src/version.c

```diff
@@ -10,32 +10,53 @@
 #include <funcapi.h>
 #include <fmgr.h>
 #include <storage/fd.h>
+#include <utils/timestamp.h>
 
 #include "fmgr.h"
 #include "compat.h"
+#include "annotations.h"
 #include "gitcommit.h"
 #include "version.h"
 #include "config.h"
 
-#define STR_EXPAND(x) #x
-#define STR(x) STR_EXPAND(x)
-
-static const char *git_commit = STR(EXT_GIT_COMMIT);
+#ifdef GIT_FOUND
```

Comment:
I think we need to provide this implementation even if the git information is not available. Otherwise the SQL function will have a missing C-implementation. (Also note that tests don't pass, but I don't know if it is for this reason or not).

Reply:
In the previous commit, the function definition was excluded if Git was not found, so this worked as expected, but after our discussions it now returns an explicit error, hence it needs to be always defined.